tests/test_custom_checkers/test_function_parameter_not_mentioned_checker.py

```diff
@@ -0,0 +1,302 @@
+import astroid
+import pylint.testutils
+from astroid import nodes
+
+from python_ta.checkers.function_parameter_not_mentioned_checker import (
+    FunctionParameterNotMentionedChecker,
+)
+
+
+class TestFunctionParameterNotMentionedChecker(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = FunctionParameterNotMentionedChecker
+
+    def setUp(self):
+        self.setup_method()
+
+    # TODO: The follwoing cases need to change, the node is no longer a function node, but instead a parameter node
+    def test_no_missing_parameter(self) -> None:
+        """Test the checker on a function with no missing parameters"""
+        src = '''
+                def f(x: int) -> int:
+                    """Return one plus x
+                    """
+                    pass
+                '''
+        mod = astroid.parse(src)
+        function_node, *_ = mod.nodes_of_class(nodes.FunctionDef)
+
+        with self.assertNoMessages():
+            self.checker.visit_functiondef(function_node)
+
+    def test_multiple_missing_parameters(self) -> None:
```

Comment:
This test name and description does not align with what's actually being tested.